jsapp/js/utils.ts

```diff
@@ -27,35 +27,49 @@ export function notify(msg: string, atype = 'success') {
   alertify.notify(msg, atype);
 }
 
+const unzonedTime = new RegExp(/^\d{4}-\d\d-\d\dT\d\d:\d\d:\d\d$/);
+
+/**
+ * Given a string like '2022-07-22T00:00:00', appends a 'Z' so it can be interpreted as UTC instead of "local".
+ */
+function presumeUtc(timeStr: string): string {
+  // Expecting like	'2022-01-01T00:00:00', or that plus a timezone offset.
+  if (unzonedTime.test(timeStr)) {
```

Comment:
I'm not sure if this regexp test is really needed, let me know what you think.
Originally I just wrote `if (timeStr.length === 19)`, but I thought this would make the intent a little more clear.

Reply:
I like that function :) And it's a perfect case for you to write some unit tests! Check out some other tests in `/jsapp/js/utils.tests.es6` - the `formatSeconds` one is pretty concise. Think about some edge cases :) 

To run tests you have to run `npm run test` in terminal. Note that sometimes these tests fail (like 1% chance) due to test suite not finding the headless chrome instace - if that happens, just re-run tests :)

Reply:
Ah, I like it too, but now I've found the built-in moment function that makes it not needed.

Reply:
By the way, we don't currently have unit tests for [these formatTime functions](https://github.com/kobotoolbox/kpi/blob/21c5b2241d0c7287728eff223de29862234a7aca/jsapp/js/utils.ts#L30-L60). To me that makes sense since we'd need to mock the local timezone (or even the current date), and since they're just moment wrappers, it doesn't seem worth the complexity. What do you think?

Reply:
It's great you've found it. And agree, no point in testing moment library functions :)